components/elm/src/dyn_subgrid/dyncropFileMod.F90

```diff
@@ -61,8 +62,9 @@ subroutine dyncrop_init(bounds, dyncrop_filename)
     !
     ! !LOCAL VARIABLES:
     integer :: num_points     ! number of spatial points
-    integer :: wtcft_shape(2) ! shape of the wtcft data
-    integer :: fertcft_shape(2) ! shape of the fertcft data
+    integer :: wtcrop_shape(2) ! Shape of hte wtcrop data   TKT
+    integer :: wtcft_shape(3) ! shape of the wtcft data      TKT
+    integer :: fertcft_shape(3) ! shape of the fertcft data    TKT
     character(len=*), parameter :: subname = 'dyncrop_init'
```

Comment:
Remove initials in comments here and throughout module

Reply:
@thorntonpe I have removed comment initials as requested. Thanks.